src/pointer-analysis/value_set_dereference.cpp

```diff
@@ -107,12 +107,12 @@ exprt value_set_dereferencet::dereference(const exprt &pointer)
         may_fail=true;
   }
 
+  exprt failure_value;
```

Comment:
I find it too non-trivial to check that `failure_value` is never used uninitialised. See comment below.